frontend/src/pages/Testbed.tsx

```diff
@@ -61,14 +63,81 @@ const logChange = (...args: unknown[]) => {
 
 /** test and example usage of formatting, elements, components, etc. */
 const TestbedPage = () => {
+  /** generate fake node data */
+  const nodes = Array(200)
+    .fill(null)
+    .map(() => ({
+      id: uniqueId(),
+      label: sample([
+        "Lbl.",
+        "Label",
+        "Long Label",
+        "Really Long Label",
+        undefined,
+      ]),
+      type: sample([
+        "gene",
+        "disease",
+        "compound",
+        "anatomy",
+        "phenotype",
+        undefined,
+      ]),
+      strength: sample([0, 0.1, 0.02, 0.003, 0.0004, 0.00005, undefined]),
+      extra: sample(["cat", "dog", "bird"]),
```

Comment:
do you need example networks | or can you pull this from one of the example molevolvr runs?

Reply:
The fake data I'm generating here should explicitly stay fake, because I've set it up to test various cases (e.g. wide range of values, looped nodes, stacked edges, etc.)..

This is just for the testbed though. I will eventually ask for sample network data from a real analysis, and I'll use that as "mock" data that will be used when running tests or available behind a flag (e.g. put `?mock=true` in the url to run the frontend with mock data and not hit the real API). At that point, we'll be able to see that real data on the Analysis results page, and we'll also be able to make visual tweaks to suit the actual data.